test/auth_oidc/test_auth_oidc.py

```diff
@@ -219,6 +227,26 @@ def test_1_6_allowed_hosts_blocked(self):
         # Close the client.
         client.close()
 
+    def test_1_7_allowed_hosts_in_connection_string_ignored(self):
+        # Create an OIDC configured client with the connection string: `mongodb+srv://example.com/?authMechanism=MONGODB-OIDC&authMechanismProperties=ALLOWED_HOSTS:%5B%22example.com%22%5D` and a Human Callback.
+        # Assert that the creation of the client raises a configuration error.
+        uri = "mongodb+srv://example.com?authMechanism=MONGODB-OIDC&authMechanismProperties=ALLOWED_HOSTS:%5B%22example.com%22%5D"
+        with self.assertRaises(ConfigurationError), warnings.catch_warnings():
+            warnings.simplefilter("ignore")
+            _ = MongoClient(
```

Comment:
Pointless assignment just for the linter?

Reply:
Yes